docs/sites.yml

```diff
@@ -7530,3 +7530,27 @@
   categories:
     - Education
     - Learning
+- title: AJ on Purr-fect Solutions
+  url: https://ajonp.com
+  main_url: https://ajonp.com
+  description: >
+    A Community of developers, creating resources for all to use!
+  categories:
+    - Education
+    - Learning
+    - Programming
+    - Web Development
+    - API
+    - Blog
+    - CMS:sanity.io
+    - Firebase
+    - SEO
+    - Netlify
+    - Pagination
+    - RSS
+    - Search
+    - Styling:PostCSS
+    - Styling:Tailwind
```

Comment:
Whitelisted site categories are listed in https://github.com/gatsbyjs/gatsby/blob/master/docs/categories.yml#L64-L117 , please remove those that are not in the list:
```
    - CMS:sanity.io
    - Firebase
    - Netlify
    - Pagination
    - RSS
	- Search
    - Styling:PostCSS
    - Styling:Tailwind
```

Reply:
Cool thank you for clarification.